tests/FakeItEasy.Specs/UnnaturalFakeSpecs.cs

```diff
@@ -59,5 +159,20 @@ public static void AnyCall(Fake<IFoo> fake, bool wasCalled)
             "Then it invokes the action"
                 .x(() => wasCalled.Should().BeTrue());
         }
+
+        public class Foo : IFoo
+        {
+            public bool WasVoidMethodCalled { get; private set; }
+
+            public virtual void VoidMethod()
+            {
+                this.WasVoidMethodCalled = true;
+            }
+
+            public virtual int MethodWithResult()

```

Comment:
No, I suppose not!
I'd started using `IFoo`s until I wanted to test `CallsBaseMethod` and then I foolishly made a class implement `IFoo`.
I've pushed another commit to remove `IFoo` as an interface of `Foo` (which I've renamed) and clean up one Code Analysis violation.
Well spotted. 
I guess @thomaslevesque [wasn't trying hard enough](https://github.com/FakeItEasy/FakeItEasy/pull/835#issuecomment-241655071). (I kid. All deficiencies in the code are my responsibility, of course.)